### PR TITLE
Add: Dekstop-app

### PR DESCRIPTION
**Pull request by RaulTrombin**
image

### CODE REVIEW

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -92,3 +97,69 @@ jobs:
...
+
+  publish-tauri:
+          needs: quick-tests
+          permissions:
+            contents: write
+          strategy:
+            fail-fast: false
+            matrix:
+              include:
+                - platform: 'macos-latest' # for Arm based macs (M1 and above).
```
Comment (patrickelectric) [inline .github/workflows/action.yml:109]: Keep in mind that macos-13 is for Intel and macos-14/macos-latest is for ARM.

Comment (RaulTrombin) [reply .github/workflows/action.yml:109]: bellow there is the args prop,
```
                - platform: 'macos-latest' # for Arm based macs (M1 and above).
                  args: '--target aarch64-apple-darwin'
                - platform: 'macos-latest' # for Intel based macs.
                  args: '--target x86_64-apple-darwin'
```

File: ping-viewer-next-desktop/.gitignore [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Logs
```
Comment (patrickelectric) [inline ping-viewer-next-desktop/.gitignore:1]: This is not necessary, can we clean it up ?

File: ping-viewer-next-desktop/src-tauri/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+[package]
+name = "ping-viewer-next-desktop"
+version = "0.1.0"
+description = "A Tauri App"
+authors = ["you"]
```
Comment (patrickelectric) [inline ping-viewer-next-desktop/src-tauri/Cargo.toml:5]: Missing authors

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: ping-viewer-next-desktop/.gitignore (hunk 2 above)
Comment (RaulTrombin) [reply ping-viewer-next-desktop/.gitignore:1]: this is the Logs default folder when we execute the desktop app with logger module, would suggest to keep it

Comment (patrickelectric) [reply ping-viewer-next-desktop/.gitignore:1]: I'm just saying to remove the unnecessary stuff, .idea, .dist-ssr, .sln

File: .github/workflows/action.yml [hunk 4]
```diff
@@ -92,3 +97,77 @@ jobs:
...
+              with:
+                projectPath: './ping-viewer-next-desktop'
+                args: ${{ matrix.args }}
+
+            - name: Upload artifacts
+              uses: actions/upload-artifact@v4.4.0
+              with:
+                name: 'ping-viewer_${{steps.tauri-build.outputs.appVersion}}_${{ matrix.label }}'
+                path: "${{ join(fromJSON(steps.tauri-build.outputs.artifactPaths), '\n') }}"
+                retention-days: 5
```
Comment (patrickelectric) [inline .github/workflows/action.yml:173]: Missing EOL

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: .github/workflows/action.yml [hunk 5]
```diff
@@ -92,3 +97,77 @@ jobs:
...
+          needs: quick-tests
+          permissions:
+            contents: write
+          strategy:
+            fail-fast: false
+            matrix:
+              include:
+                - platform: 'macos-latest'
+                  args: '--target aarch64-apple-darwin'
+                  label: 'Mac-M1'
```
[the remark this answers is not in the document]
Comment (RaulTrombin) [reply .github/workflows/action.yml:111]: @patrickelectric

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**